frontend/src/App.vue

```diff
@@ -2,10 +2,12 @@
 import Scoreboard from './components/Scoreboard.vue';
 import ScoreKeeper from "@/components/scorekeeping/ScoreKeeper.vue";
```

Comment:
I think we'll need to define a style for how we're referencing components (using `@/` vs `./`

Reply:
Agreed, I don't really care which way, my IDE just auto-completes to `@/`, I'm sure I can figure out how to change that if we decide otherwise though.